c++/src/kj/async-unix.c++

```diff
@@ -1065,7 +1065,11 @@ void UnixEventPort::captureChildExit() {
 void UnixEventPort::wake() const {
   // Trigger our user event.
   struct kevent event;
+#if defined(NOTE_TRIGGER)
   EV_SET(&event, 0, EVFILT_USER, 0, NOTE_TRIGGER, 0, nullptr);
+#elif defined(EV_TRIGGER)
+  EV_SET(&event, 0, EVFILT_USER, EV_TRIGGER, 0, 0, nullptr);
+#endif
```

Comment:
If neither NOTE_TRIGGER nor EV_TRIGGER is defined, this code will still compile but produce undefined behavior.

```suggestion
#else
#error "neither NOTE_TRIGGER nor EV_TRIGGER is defined; we need at least one of these"
#endif
```